albumentations/augmentations/geometric/functional.py

```diff
@@ -597,3 +592,171 @@ def safe_rotate_enlarged_img_size(angle: float, rows: int, cols: int):
+            maps have values in the range ``(0.0, 1.0]`` with ``1.0`` denoting
+            exactly the position of the respective keypoint.
+
+    Returns:
+        (H,W,N) ndarray
+            A ``float32`` array containing ``N`` distance maps for ``N``
+            keypoints. Each location ``(y, x, n)`` in the array denotes the
+            euclidean distance at ``(y, x)`` to the ``n``-th keypoint.
+            If `inverted` is ``True``, the distance ``d`` is replaced
+            by ``d/(d+1)``. The height and width of the array match the
+            height and width in ``KeypointsOnImage.shape``.
+    """
+    distance_maps = np.zeros((height, width, len(keypoints)), dtype=np.float32)
+
+    yy = np.arange(0, height)
+    xx = np.arange(0, width)
+    grid_xx, grid_yy = np.meshgrid(xx, yy)
+
+    for i, (x, y) in enumerate(keypoints):
+        distance_maps[:, :, i] = (grid_xx - x) ** 2 + (grid_yy - y) ** 2
+
+    distance_maps = np.sqrt(distance_maps)
+    if inverted:
+        return 1 / (distance_maps + 1)
+    return distance_maps
+
+
+def from_distance_maps(
+    distance_maps: np.ndarray,
+    inverted: bool,
+    if_not_found_coords: Optional[Union[Sequence[int], dict]],
+    threshold: Optional[float] = None,
+) -> List[Tuple[float, float]]:
+    """Convert outputs of ``to_distance_maps()`` to ``KeypointsOnImage``.
+    This is the inverse of `to_distance_maps`.
+
+    Args:
+        distance_maps (np.ndarray): The distance maps. ``N`` is the number of keypoints.
+        inverted (bool): Whether the given distance maps were generated in inverted mode
+            (i.e. :func:`KeypointsOnImage.to_distance_maps` was called with ``inverted=True``) or in non-inverted mode.
+        if_not_found_coords (tuple, list, dict or None, optional):
+            Coordinates to use for keypoints that cannot be found in `distance_maps`.
+
+            * If this is a ``list``/``tuple``, it must contain two ``int`` values.
+            * If it is a ``dict``, it must contain the keys ``x`` and ``y`` with each containing one ``int`` value.
+            * If this is ``None``, then the keypoint will not be added.
+        threshold (float): The search for keypoints works by searching for the
+            argmin (non-inverted) or argmax (inverted) in each channel. This
+            parameters contains the maximum (non-inverted) or minimum (inverted) value to accept in order to view a hit
+            as a keypoint. Use ``None`` to use no min/max.
+        nb_channels (None, int): Number of channels of the image on which the keypoints are placed.
+            Some keypoint augmenters require that information. If set to ``None``, the keypoint's shape will be set
+            to ``(height, width)``, otherwise ``(height, width, nb_channels)``.
+    """
+    assert (
+        distance_maps.ndim == 3
+    ), f"Expected three-dimensional input, got {distance_maps.ndim} dimensions and shape {distance_maps.shape}."
```

Comment:
I think it is better to use a `ValueError` exception instead of `assert` for these errors.

Reply:
Good point, thank you! Fixed.